lib/ex_doc.ex

```diff
@@ -112,9 +136,30 @@ defmodule ExDoc do
   end
 
   # Helpers
+  defp nilify_options(options) do
+    for {field, value} <- options do
+      case value do
+        "" -> {field, nil}
+        _  -> {field, value}
+      end
+    end
+  end
+
+  defp normalize_option(options, field, default) do
```

Comment:
This function can be written as: `options[field] || default`. Which is how it was being used before.